src/styles/blocks/minute.scss

```diff
@@ -0,0 +1,23 @@
+.minute {
+  height: 100px;
+  width: 8px;
+  background-color: aquamarine;
+  border-radius: 8px;
+  position: absolute;
+  top: 100px;
+  animation-name: rotateMinute;
```

Comment:
You can write it in one line, like this:
animation: rotateMinute 3600s linear .....
for aesthetics)))